src/Processors/Formats/Impl/NativeORCBlockInputFormat.h

```diff
@@ -124,11 +130,20 @@ class ORCColumnToCHColumn
         Chunk & res, NameToColumnPtr & name_to_column_ptr, size_t num_rows, BlockMissingValues * block_missing_values = nullptr);
 
 private:
+    ColumnWithTypeAndName readColumnFromORCColumn(
+        const orc::ColumnVectorBatch * orc_column,
+        const orc::Type * orc_type,
+        const std::string & column_name,
+        bool inside_nullable,
+        DataTypePtr type_hint = nullptr) const;
+
     const Block & header;
     /// If false, throw exception if some columns in header not exists in arrow table.
     bool allow_missing_columns;
     bool null_as_default;
     bool case_insensitive_matching;
+    bool dictionary_as_low_cardinality;
+    std::unique_ptr<orc::StripeInformation> stripe_info;
```

Comment:
It seems that the `stripe_info` member is not used in any member function of class `ORCColumnToCHColumn`.

Reply:
I'll fix it.